x-pack/plugins/ml/server/models/data_frame_analytics/analytics_manager.ts

```diff
@@ -55,12 +66,21 @@ export class AnalyticsManager {
       const models = await this.getAnalyticsModels();
       this.inferenceModels = models;
     } catch (error) {
-      // TODO: bubble up this error?
       // eslint-disable-next-line
       console.error('Unable to fetch inference models', error);
     }
   }
 
+  async setJobStats() {
+    try {
+      const jobStats = await this.getAnalyticsStats();
+      this.jobStats = jobStats;
+    } catch (error) {
+      // eslint-disable-next-line
+      console.error('Unable to fetch job stats', error);
```

Comment:
do we want it to fail silently? I guess worth showing a toast notification instead.

Reply:
I think it might be worth refactoring a bit to allow an array of errors to be returned - that way we can return as much data as possible to the user and then let them know what failed. I will add this as a follow up item if that sounds good 👌